packages/bpk-component-banner-alert/src/BpkBannerAlertInner.js

```diff
@@ -258,7 +258,7 @@ BpkBannerAlertInner.defaultProps = {
   onExpandToggle: null,
 
   // Only relevant when configuration == CONFIGURATION.DISMISSABLE
-  dismissButtonLabel: null,
+  dismissButtonLabel: '',
```

Comment:
Changed to align with above

Reply:
If it can't be null, does it even need to be in `defaultProps`? i.e. what happens when you remove this line – do things break?